### ISSUE TITLE
Bib Bonn wechselt von GENIOS zu Nexis Uni

### ISSUE THREAD

**Issue by bromanoff** [stefanw/bibbot#242]
Hallo zusammen,

auf Anfrage bestätigte mir eine Mitarbeiterin der Stadtbibliothek Bonn, dass der Zugang zu GENIOS Ende 2022 eingestellt wurde. Seit 2023 beinhaltet das ([Onlineangebot](https://www.bonn.de/themen-entdecken/bildung-lernen/digital.php)) den Zugang zu Nexis Uni (https://login.bonn.idm.oclc.org/login).

Beim aufrufen eines Artikels öffnet BibBot über die Stadtbibliothek Bonn noch die Pressedatenbank GENIOS. Gibt es die Möglichkeit hier standardmäßig Nexis Uni aufzurufen?

**Comment by stefanw** [comment]
Ja, allerdings kann ich ohne Bib-Bonn-Konto keine exakte Implementierung vornehmen. Gerne per E-Mail vertraulich sharen.

**Comment by sleepy-nols** [comment]
can be closed, nexis uni is working with bib bonn
